packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -267,11 +267,27 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
           rendering: componentRendering,
         };
 
-        const rendered = React.createElement<{ [attr: string]: unknown }>(
+        let rendered = React.createElement<{ [attr: string]: unknown }>(
           component as React.ComponentType,
           this.props.modifyComponentProps ? this.props.modifyComponentProps(finalProps) : finalProps
         );
 
+        const type = rendered.props.type === 'text/sitecore' ? rendered.props.type : '';
+        if (!isEmpty) {
```

Comment:
@art-alexeyenko Add a comment here explaining that components are not wrapped in _Chromes Edit Mode_